plugins/koterm.koplugin/main.lua

```diff
@@ -0,0 +1,714 @@
+--[[
+This plugin provides a terminal emulator (VT52 (+some ANSI))
+]]
+
+local Device = require("device")
+
+-- grantpt and friends are necessary (introduced on Android in API 21).
+-- So sorry for the Tolinos with (Android 4.4.x)
+if Device:isAndroid() then
+    local A, android = pcall(require, "android")  -- luacheck: ignore
+    local api = android.app.activity.sdkVersion
+    if api < 21 then
+        return { disabled = true }
+    end
+end
+
+local Dispatcher = require("dispatcher")
+local Font = require("ui/font")
+local InfoMessage = require("ui/widget/infomessage")
+local InputDialog = require("ui/widget/inputdialog")
+local MultiConfirmBox = require("ui/widget/multiconfirmbox")
+local ScrollTextWidget = require("ui/widget/scrolltextwidget")
+local SpinWidget = require("ui/widget/spinwidget")
+local UIManager = require("ui/uimanager")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+local TermInputText = require("terminputtext")
+local TextWidget = require("ui/widget/textwidget")
+local T = require("ffi/util").template
+local bit = require("bit")
+local logger = require("logger")
+local _ = require("gettext")
+
+local ffi = require("ffi")
+local C = ffi.C
+
+-- for terminal emulator
+ffi.cdef[[
+static const int SIGTERM = 15;
+
+int grantpt(int fd) __attribute__((nothrow, leaf));
+int unlockpt(int fd) __attribute__((nothrow, leaf));
+char *ptsname(int fd) __attribute__((nothrow, leaf));
+pid_t setsid(void) __attribute__((nothrow, leaf));
+
+static const int TCIFLUSH = 0;
+int tcdrain(int fd) __attribute__((nothrow, leaf));
+int tcflush(int fd, int queue_selector) __attribute__((nothrow, leaf));
+]]
+
+--[[
+local ctrl_c = "\003"
+local ctrl_f = "\006" -- right
+local ctrl_b = "\002" -- left
+local ctrl_n = "\014" -- down
+local ctrl_p = "\016" -- up
+local ctrl_x = "\024"
+local ctrl_z = "\026"
+]]
+
+local esc = "\027"
+
+local CHUNK_SIZE = 80 * 40 -- max. nb of read bytes (reduce this, if taps are not detected)
+
+--- @todo: A better message here would be helpful
+local about_text = _([[
+KOTerm is a terminal emulator, which starts a shell (command prompt).
+
+You can enter various low level commands here.
+
+You can place commands, which are executed on start in './plugins/koterm.koplugin/scripts/profile.user' and aliases (shortcuts) to command you use often in './plugins/koterm.koplugin/scripts/aliases'.]])
```

Comment:
I'd suggest something like:
```suggestion
Commands to be executed on start can be placed in './plugins/koterm.koplugin/scripts/profile.user'.
Aliases (shortcuts) to frequently used commands can be placed in './plugins/koterm.koplugin/scripts/aliases'.]])
```